[Exchange](custom functions) update for stock api change (#1090)
OfficeDev/office-js-docs-pr: docs/tutorials/excel-tutorial-create-custom-functions.md

## Create a custom function that requests data from the web

> Integrating data from the Web is a great way to extend Excel through custom functions. Next you’ll create a custom function named `stockPrice` that gets a stock quote from a Web API and returns the result to the cell of a worksheet. The code calls the IEX Trading API which you'll need to create an account for.

Reviewer: ```suggestion
Integrating data from the Web is a great way to extend Excel through custom functions. Next you’ll create a custom function named `stockPrice` that gets a stock quote from a Web API and returns the result to the cell of a worksheet. 

> [!NOTE]
> The following code requests a stock quote using the IEX Trading API. Before you can run the code, you'll need to [create a free account with IEX Cloud](https://iexcloud.io/) so that you can get the API token that's required in the API request.  
```

    * @returns {number} The current stock price.
    */
    function stockPrice(ticker) {

>         //Note: you will need to register for an account to get a token on https://iexcloud.io/

Reviewer: ```suggestion
        //Note: In the following line, replace <YOUR_TOKEN_HERE> with the API token that you've obtained through your IEX Cloud account.
```

                return;
            }

>             //Note: you will need to register for an account to get a token on https://iexcloud.io/

Reviewer: ```suggestion
            //Note: In the following line, replace <YOUR_TOKEN_HERE> with the API token that you've obtained through your IEX Cloud account.
```
